deploy/cloudformation/iiif-service.yml

```diff
@@ -167,8 +160,7 @@ Resources:
     DependsOn: LoadBalancerRule
     Properties:
       ServiceName: !Sub '${AWS::StackName}-ImageService'
-      Cluster:
-        Fn::ImportValue: !Join [':', [!Ref InfrastructureStackName, 'ClusterName']]
```

Comment:
Could removing these as imports could lead to a state where the service is referencing something that no longer exists, or will AWS enforce the existence of the cluster if you attempt to delete it without first removing the services within?

Reply:
As long as tasks are active within it, you cannot delete it.